env/AWSPW.env

```diff
@@ -14,8 +14,8 @@ fi
 
 step=$1
 
-export launcher="mpiexec.hydra"
-export mpmd_opt=""
+export launcher="srun --mpi=pmi2 -l"
+export mpmd_opt="--distribution=block:block --hint=nomultithread --cpus-per-task=1"
```

Comment:
So far it runs with forecast model.
as we define:
    export APRUN_UFS="${launcher} -n ${ntasks} ${mpmd_opt}"

for each step (fcst, atmos-products), we can remove it from some step if we do not need it.

Reply:
I don't think that you would want MPMD options to be sent to the UFS since the forecast job does not have an MPMD task in it.  If this change is only needed for the UFS, then I would suggest adding `--distribution=block:block --hint=nomultithread --cpus-per-task=1` to `APRUN_UFS` directly rather than assigning it globally here.

Reply:
switch mpmd_opt to "" as suggested.